react.js

```diff
@@ -8,9 +8,9 @@ module.exports = {
     // So we can appropriately have our base rules override airbnb's, we'll need to extend the react rules the same way
     // eslint-config-airbnb does internally.
     // Reference: https://github.com/airbnb/javascript/blob/eslint-config-airbnb-v17.1.0/packages/eslint-config-airbnb/index.js
-    'airbnb/rules/react',
-    'airbnb/rules/react-a11y',
-  ],
+    'eslint-config-airbnb/rules/react',
```

Comment:
@minhtule, even though when using `yarn link` locally worked when resolving the airbnb library, post version bump of eslint-config-ebates, eslint would fail when trying to resolve the airbnb library. going to try this out to see if it fixes that issue